Without a Git configuration git(1) exits with a non-zero exit code
which, in turn, causes a failure of the entr test suite.

diff -upr entr-5.3.orig/system_test.sh entr-5.3/system_test.sh
--- entr-5.3.orig/system_test.sh	2023-01-21 10:50:35.430697263 +0100
+++ entr-5.3/system_test.sh	2023-01-21 10:51:10.478073569 +0100
@@ -250,6 +250,8 @@ try "exec single utility when an entire
 		cp /usr/include/*.h $tmp/
 		cd $tmp
 		git init -q
+		git config user.email "you@example.com"
+		git config user.name "Your Name"
 		git add *.h
 		git commit -m "initial checkin" -q
 		for f in `ls *.h | head`; do
